internal/pkg/manifest/lb_web_svc.go

```diff
@@ -183,6 +183,7 @@ func (s LoadBalancedWebService) ApplyEnv(envName string) (WorkloadManifest, erro
 // RoutingRule holds the path to route requests to the service.
 type RoutingRule struct {
```

Comment:
Is it possible to also update this method? 🥺  https://github.com/aws/copilot-cli/blob/479f892092b3979ef3cb961a0e1c1ceba89fc7de/internal/pkg/manifest/validate.go#L495

such that:
```go
var httpProtocolVersions = []string{"grpc", " http1", "http2"} // global variable.

// in Validate method:
if r.ProtocolVersion != nil {
   if !contains(strings.ToLower(*r.ProcolVersion), httpProtocolVersions) {
      return fmt.Errorf(`"protocol" field value '%s' must be one of %s`, *r.ProtocolVersion, english.WordSeries(httpProtocolVersions, "or"))
   }
}
```

Reply:
OK. I will try.